elotl/nahuatl/morphology.py

```diff
@@ -243,178 +44,5 @@ def __init__(self, tokeniser=None):
-			i += 1
-
-		# Sort the analyses by weight, higher weight = worse
-		analyses.sort(key=lambda x: x[1])
-
-		converted = []
-		for (analysis, weight) in analyses:
-			converted.append((self._convert_analysis(analysis), weight))
-
-		return converted
-
-	def analyse(self, text, tokenise=False, normalise=False, max_analyses=None):
-		"""
-		An analyse function that can take either a string with a tokeniser,
-		a pre-tokenised list or a pre-tokenised string. If it is passed a
-		tokeniser the tokeniser is first run and then the list is analysed,
-		if passed a list, the list is analysed, if passed a pre-tokenised
-		list then the list is split on space and then analysed.
-
-		Parameters
-		----------
-		text: str or list
-			Input text/sentence. It can be either a string or a list. It
-			should be pre-tokenised, alternatively see below.
-
-		tokenise: bool
-			Should tokenisation be performed on the input prior to analysis?
-
-		normalise: bool
-			Should normalised forms be added to the list of alternatives?
-
-		max_analyses: int
-			The maximum number of analyses that should be returned, these
-			will be returned in order of weight, lowest weight first. If
-			None, return all analyses.
-
-		Returns
-		----------
-		list
-			List of Token objects.
-
-		Examples
-		----------
-
-		>>> a.analyse('a b c')
-		[<Token "a" (0)>, <Token "b" (0)>, <Token "c" (0)>]
-		>>> a.analyse(['a', 'b', 'c'])
-		[<Token "a" (0)>, <Token "b" (0)>, <Token "c" (0)>]
-		>>> a.analyse('ab, c', tokenise=True)
-		[<Token "a" (0)>, <Token "b" (0)>, <Token "," (0)>, <Token "c" (0)>]
-		"""
-
-		tokens = []
-		wordforms = []
-		if not tokenise:
-			if type(text) == str:
-				wordforms = text.split(' ')
-			elif type(text) == list:
-				wordforms = text
-		else:
-			wordforms = self.tokenise(text)
-
-		normaliser = Normaliser('ack')
-
-		for wordform in wordforms:
-			alternatives = [wordform.lower()]
-			if normalise:
-				alternatives += [normaliser.normalize(wordform),
-					normaliser.normalize(wordform).lower()]
-			analyses = self._analyse_token(wordform, alternatives)
-			token = Token(wordform, analyses, max_analyses)
-			tokens.append(token)
-
-		return tokens
-
-	def analyze(self, text, tokenize=False, normalize=False, max_analyses=None):
-		"""
-		Convenience alias of analyse() with alternative spelling.
-
-		Parameters
-		----------
-		text: str or list
-			Input text/sentence. It can be either a string or a list. It
-			should be pre-tokenised, alternatively see below.
-
-		tokenize: bool
-			Should tokenisation be performed on the input prior to analysis?
-
-		max_analyses: int
-			The maximum number of analyses that should be returned, these
-			will be returned in order of weight, lowest weight first. If
-			None, return all analyses.
-
-		Returns
-		----------
-		list
-			List of Token objects.
-		"""
-		return self.analyse(text, tokenize, normalize, max_analyses)
-
-# Convenience alias for Analyser to Analyzer
-Analyzer = Analyser
```

Comment:
We should keep this in the Nahuatl morphology file.

Reply:
Oops, missed taking those out. Now restored.